src/redux/watchList/reducer.js

```diff
@@ -0,0 +1,61 @@
+import { FETCH_WATCHLIST_SUCCESS, POST_WATCHLIST_SUCCESS } from './actionCreator';
+
+const watchlist1 = {
+  id: 1,
+  name: 'A share',
+  stocks: [],
+};
+
+const watchlist2 = {
+  id: 2,
+  name: 'NSDQ',
+  stocks: [],
+};
+
+const initalState = {
+  watchlists: [watchlist1, watchlist2],
+  stocks: [
```

Comment:
stocks are part of the watchlist, they are not separated entities. how would you do this when we will have real watchlists with different stocks?